### ISSUE TITLE
Documentation for the novices

### ISSUE THREAD

**Issue by JustinTBrown** [burnbright/silverstripe-importexport#27]
It would be rad if the documentation, though thorough, was a bit more accessible for more junior developers. Looking at your example code snippets, it's not entirely clear where I need to be using them to add the functionality of this module.

**Comment by jeffwhitfield** [comment]
Yeah, I have to admit, this is very, very confusing. I've set it to replace all legacy ModelAdmin importers which works fine. However, I have instances where some fields are not available to be mapped. 

For instance, I want the 'Created' field to show up so that I can map a date to it...but it's not showing up at all in the drop-downs. I'd like to know how to update my ModelAdmin instance so that I can easily update the mappable fields. Can't be hard, right?

Also, I have a model with a one-to-many relation that doesn't show up at all. The one-to-one relation shows up...just not the one-to-many relation. So I would like to be able to update the mappable fields for this ModelAdmin as well. 

Basically, just need a good example of how to take an existing ModelAdmin that has been replaced already with the new ModelAdmin importer and update it with customizations and overrides so that it accurate reflects the model. Kinda lost. :P

**Comment by helenclarko** [comment]
@jeffwhitfield and @JustinTBrown,

I'm also looking for some help here.

I am using this add-on as an easy means of importing products into SilverShop. The import works for the most part, but feilds fail to show for the likes of ParentID, which is needed to set categories.
The documentation here is rather complicated and doesn't give much of an example for where the linking code should be placed.

Any help would be appreciated.

**Comment by mlewis-everley** [comment]
Agree with the above, This module is awesome, but I am trying to work out how to manipulate GridFieldImporter to add callbacks for certain column heads.

Maybe just a complete code example at the end of the docs, using all the individual elements of the docs would be enough?
